fastlane/lib/fastlane/commands_generator.rb

```diff
@@ -58,7 +58,12 @@ def run
         c.description = 'Helps you with your initial fastlane setup'
 
         c.action do |args, options|
-          Fastlane::Setup.new.run
+          if args[0] == 'beta' && FastlaneCore::Feature.enabled?('FASTLANE_ENABLE_CRASHLYTICS_BETA_INITIALIZATION')
```

Comment:
cool, that makes sense, then should [this line](https://github.com/fastlane/fastlane/pull/5201/files/14ea820d33ee8d4bc18bb5680b518fe3f6ea5221#diff-6d9cdfc0945e56dd3efea6ab7e8d11aeR2) be updated to reflect that?